src/resolvers/UserTag/parentTag.ts

```diff
@@ -0,0 +1,8 @@
+import { UserTagResolvers } from "../../types/generatedGraphQLTypes";
+import { OrganizationTagUser } from "../../models";
+
+export const parentTag: UserTagResolvers["parentTag"] = async (parent) => {
+  return await OrganizationTagUser.findOne({
```

Comment:
Here check if `parent.parentTagId` is null. If it is just return null, you don't have to make a database request.

Reply:
Got it.